nats.go

```diff
@@ -146,6 +146,8 @@ var (
 	ErrInvalidJSAck                 = errors.New("nats: invalid jetstream publish response")
 	ErrMultiStreamUnsupported       = errors.New("nats: multiple streams are not supported")
 	ErrStreamNameRequired           = errors.New("nats: stream name is required")
+	ErrNoStream                     = errors.New("nats: no stream found")
+	ErrNoConsumer                   = errors.New("nats: no consumer found")
```

Comment:
Can we change to this so that it is backwards compatible in case matching against the string?

```suggestion
	ErrNoConsumer                   = errors.New("nats: consumer not found")
```

Thanks!

Reply:
Will make this change now!

Reply:
I think there are tests that depend on the error being `consumer not found`, not `nats: consumer not found`.
Here's one that's failing.
```
js_test.go:3906: Expected consumer not found error, got: nats: consumer not found
```

I think we have to either drop the prefix or update the test.

Maybe based on https://github.com/nats-io/nats.go/pull/760#discussion_r657423111, we drop the prefix?

Reply:
I noticed this and was going in to change from a string compare to use errors.Is() with the new error types in the tests. I added the prefix to keep consistency with the rest of the sentinel errors. For backwards compatibility sake I could remove the prefix

Reply:
Making the change to remove the prefix for backwards compatibility. All the effected tests are passing locally. This should close issue #751 

This is my first real contribution to anything open source, so I appreciate how fast you all have responded. I'm a huge fan of the NATS project and hope I can contribute more in the future as well.

Reply:
Awesome! That's great to hear! Thank you very much for the contribution! 🎉